### DISCUSSION TITLE
anyone who has had success with aws fargate?

### DISCUSSION THREAD

**Discussion by leehavin** [dotnet/dotnet-monitor#5372] [Q&A] [answered]
This seems to be very difficult to succeed, I have tried many times, process empty list, please friends, there is no relevant configuration examples

[a comment by an automated account is not shown]

**Reply by fuyfuy** [reply, marked as answer]
@leehavin have you checked this blog post? It's pretty detailed https://www.mytechramblings.com/posts/profiling-a-net-app-on-aws-ecs-fargate-with-dotnet-monitor/
